addons/mail/static/src/core/messaging_service.js

```diff
@@ -126,6 +126,7 @@ export class Messaging {
             this.insertCannedResponse(code);
         });
         this.store.hasLinkPreviewFeature = data.hasLinkPreviewFeature;
+        this.store.hasTenorFeature = data.hasTenorFeature;
```

Comment:
I feel like `hasTenorFeature` is a little bit confusing, maybe more into the point, like `GifFeature` or at least `TenorGifFeature`

Reply:
`GifFeature` seams generic and we only support Tenor.
`TenorGifFeature` feels better but kind of useless since tenor only do gif stuff? I don't know...

Reply:
haha, the problem is some people don't know what is Tenor (like me at first, so gif describes the usgae of it.)